packages/web3-providers-http/package.json

```
@@ -1,7 +1,7 @@
 {
 	"name": "web3-providers-http",
 	"version": "4.0.0-alpha.1",
-	"description": "HTTP provider for Web3 4.x.x",
+	"description": "Module to handle web3 RPC connections over IPC sockets.",
```

this package json is in http package
```suggestion
	"description": "HTTP provider for Web3 4.x.x",
```